rusk/src/lib/node/vm/config.rs

```diff
@@ -123,11 +134,17 @@ impl Config {
             .feature(feature::FEATURE_ABI_PUBLIC_SENDER)
             .map(|activation| block_height >= activation)
             .unwrap_or_default();
+        let blob_enable = self
```

Comment:
```suggestion
        let with_blob = self
```
to be consistent with the other feature